experimental-design/README.adoc

```diff
@@ -141,5 +141,70 @@ Some search engines as MaxQuant use the Fraction Group to perform better statist
 - Comment [Fraction Group]: Some Quantitative tools use the Fraction Group to know which fractions belong together. In MaxQuant the Fraction Group is called "Experiment".
 
 
+[[encoding-MSRun-technical-details]]
+== MSRun technical details properties
 
+We RECOMMEND to encode some of the technical parameters of the mass spectrometry experiment as Comments (https://www.ebi.ac.uk/arrayexpress/help/creating_a_sdrf.html[Check what is a Comment in SDRF]) including the following parameters:
```

Comment:
lower case 'c' in comment

Reply:
Commet is the way that SDRF has to encode additional parameters. SDRF is the Sample metadata file format for RNA-Seq data. Is also the way that thousands of experiments are annotated in ArrayExpress. See an example https://github.com/PRIDE-Archive/pride-metadata-standard/blob/master/experimental-design/examples/E-PROT-23.sdrf.txt